### ISSUE TITLE
selector-type-no-unknown returns false positives for keyframe selectors in mixins

### ISSUE THREAD

**Issue by epmatsw** [stylelint/stylelint#1383]
With stylelint 6.5.1 and stylelint-config-standard 8.0.0, the `selector-type-no-unknown` appears to trigger a false-positive on keyframe % selectors inside of mixins. Admittedly, this is a super edge case.

Example code:
```
@include keyframes(identifier) {
    0% {
        opacity: 0;
        @include transform( 'scale(0.95)');
    }

    80% {
        opacity: 1;
    }

    100% {
        @include transform( 'scale(1)');
        opacity: 1;
    }
}
```

This triggers 3 errors, when I would expect 0.

**Comment by doberkofler** [comment]
We have the same problem.

**Comment by nikolas6** [comment]
With stylelint 6.7.0, it doesn't like the keyframe with decimal value like "50.001%".

```css
@keyframes loadingAnimation1 {
  0% {
    transform: translate3d(0, 0, 0);
  }
  50% {
    transform: translate3d(100%, 0, 0);
  }
  50.001% {
    transform: translate3d(-100%, 0, 0);
  }
  100% {
    transform: translate3d(0, 0, 0);
  }
}
```

**Comment by davidtheclark** [comment]
@nikolas6 if you'd like to file an issue please go for it.

**Comment by nikolas6** [comment]
submitted as [https://github.com/stylelint/stylelint/issues/1513](https://github.com/stylelint/stylelint/issues/1513)
